### PR TITLE
Bug 1880117: Fix Protractor flake: display Global Configuration page, click Explore Console API in dropdown link

### PR DESCRIPTION
**Pull request by dtaylor113**
Saw this flake in two of my open PRs, also seems to be happening pretty consistently [search](https://search.ci.openshift.org/?search=display+Global+Configuration+page%2C+click+Explore+Console+API+in+dropdown+link&maxAge=48h&context=2&type=junit&name=&maxMatches=5&maxBytes=20971520&groupBy=job)

### CODE REVIEW

File: frontend/integration-tests/views/cluster-settings.view.ts [hunk 1]
```diff
@@ -16,3 +16,6 @@ export const clusterOperatorResourceLink = $('[data-test-id="console"]');
 export const globalConfigResourceLink = $('[data-test-id="Console"]');
 export const clusterResourceDetailsTitle = $('[data-test-id="resource-title"]');
 export const globalConfigDetailsTitle = $('[data-test-id="api-explorer-resource-title"]');
+export const globalConfigDetailsTitleIsLoaded = browser.wait(
+  until.presenceOf(globalConfigDetailsTitle),
+);
```
Comment (spadgett) [inline frontend/integration-tests/views/cluster-settings.view.ts:21]: Doesn't this need to be a function?

```suggestion
export const globalConfigDetailsTitleIsLoaded = () => browser.wait(
  until.presenceOf(globalConfigDetailsTitle),
);
```

File: frontend/integration-tests/tests/cluster-settings.scenario.ts [hunk 2]
```diff
@@ -95,6 +95,7 @@ describe('Cluster Settings', () => {
     expect(crudView.actionForLabel('Explore Console API').isDisplayed()).toBe(true);
     await crudView.actionForLabel('Explore Console API').click();
 
+    await clusterSettingsView.globalConfigDetailsTitleIsLoaded;
     expect(clusterSettingsView.globalConfigDetailsTitle.isDisplayed()).toBe(true);
```
Comment (spadgett) [inline frontend/integration-tests/tests/cluster-settings.scenario.ts:99]: This check isn't needed if we use `browser.wait` above.

**Review by spadgett — COMMENTED** [review]
Thanks @dtaylor113 

I'm a little unsure about the fix since the title appears to be there in the screenshot. For instance, in this run: https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_console/6633/pull-ci-openshift-console-master-e2e-gcp-console/1306188411899482112/artifacts/e2e-gcp-console/gui_test_screenshots/1051fa6fc0de148e84f8b6aad4600872.png

File: frontend/integration-tests/views/cluster-settings.view.ts (hunk 1 above)
Comment (dtaylor113) [reply frontend/integration-tests/views/cluster-settings.view.ts:21]: arrg..didn't push latest

File: frontend/integration-tests/tests/cluster-settings.scenario.ts (hunk 2 above)
Comment (dtaylor113) [reply frontend/integration-tests/tests/cluster-settings.scenario.ts:99]: Which `browser.wait` are you refering too?

Comment (dtaylor113) [reply frontend/integration-tests/tests/cluster-settings.scenario.ts:99]: Are you saying I don't need the `await ...`?

Comment (spadgett) [reply frontend/integration-tests/tests/cluster-settings.scenario.ts:99]: No, I'm saying you don't need the `expect` on line 99. You've already confirmed the title is there on the line above.

**Comment by dtaylor113** [comment]
> I'm a little unsure about the fix since the title appears to be there in the screenshot.

Yes, me too, but the failure is:
`NoSuchElementError: No element found using locator: By(css selector, [data-test-id="api-explorer-resource-title"])`, but it does appear to be the correct selector:
image
So I'm not sure if the screenshot is accurate, so testing to see if a `browser.wait` is needed.

File: frontend/integration-tests/tests/cluster-settings.scenario.ts (hunk 2 above)
Comment (dtaylor113) [reply frontend/integration-tests/tests/cluster-settings.scenario.ts]: ah, ok, removed.

**Comment by spadgett** [comment]
It might be possible the heading appeared in between the check and screenshot.

File: frontend/integration-tests/tests/cluster-settings.scenario.ts [hunk 3]
```diff
@@ -95,7 +95,7 @@ describe('Cluster Settings', () => {
     expect(crudView.actionForLabel('Explore Console API').isDisplayed()).toBe(true);
     await crudView.actionForLabel('Explore Console API').click();
 
-    expect(clusterSettingsView.globalConfigDetailsTitle.isDisplayed()).toBe(true);
+    await clusterSettingsView.globalConfigDetailsTitleIsLoaded;
```
Comment (spadgett) [inline frontend/integration-tests/tests/cluster-settings.scenario.ts:98]:
```suggestion
    await clusterSettingsView.globalConfigDetailsTitleIsLoaded();
```

**Review by spadgett — CHANGES_REQUESTED** [review]

File: frontend/integration-tests/tests/cluster-settings.scenario.ts (hunk 3 above)
Comment (dtaylor113) [reply frontend/integration-tests/tests/cluster-settings.scenario.ts]: duh, fixed, thanks

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**